server/admin/reports.js

```diff
@@ -0,0 +1,14 @@
+import mongo from '../util/mongo.js'
+
+export async function getLatestChanges() {
```

Comment:
La fonction ne fait pas vraiment ce qu'elle dit, elle renvoie les projets, pas les changements. Par ailleurs ce serait plus efficace d'avoir un paramètre `since` qui indique jusqu'à quand on recule pour l'appel. Ça ajoute des possibilités à l'API et ça évite de hard-coder une plage de temps.